bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/state/PowermaxState.java

```diff
@@ -214,6 +276,80 @@ public void updateZoneInfo(int zoneIdx, int zoneInfo) {
         this.updatedZoneInfos.put(zoneIdx, zoneInfo);
     }
 
+    // This is an attempt to add persistence to an otherwise (mostly) stateless class.
+    // All of the other values are either present or null, and it's easy to build a
+    // delta state based only on which values are non-null. But these system events
+    // are different because each event can be set by one message and cleared by a
+    // later message. So to preserve the semantics of the state class, we'll keep a
+    // queue of incoming changes, and apply them only when the delta state is resolved.
+
+    public boolean hasActiveAlertsQueued() {
+        return !activeAlertQueue.isEmpty();
+    }
+
+    public String getActiveAlerts() {
+        if (activeAlertList == null) {
+            return null;
+        }
+
+        if (activeAlertList.isEmpty()) {
+            return "None";
+        }
```

Comment:
Please return "None" when `activeAlertList` is null too, it allows having the channel initialized to "None" after binding startup.